### PR TITLE
Issue78: Follow-up

### PR DESCRIPTION
**Pull request by DanilYachmenev**
Follow-up from https://github.com/sorrentum/sorrentum/pull/99

Requested nits are implemented, except for an additional unit test (filed https://github.com/sorrentum/sorrentum/issues/109 for it)

### CODE REVIEW

File: defi/dao_cross/dao_cross.py [hunk 1]
```diff
@@ -28,15 +29,17 @@ def match_orders(
...
+    hdbg.dassert_lt(0, len(orders))
+    hdbg.dassert_container_type(orders, list, ddacrord.Order)
+    hdbg.dassert_lt(0, clearing_price)
     # Build buy and sell heaps.
     buy_heap = []
     sell_heap = []
     # Push orders to the heaps based on the action type and filtered by limit price.
     for order in orders:
-        # TODO(Dan): Think of more asserts to check if orders are compatible.
         hdbg.dassert_type_is(order, ddacrord.Order)
```
Comment (PomazkinG) [inline defi/dao_cross/dao_cross.py:42]: nit: the code at line 35 checks the same thing

File: defi/dao_cross/order.py [hunk 2]
```diff
@@ -63,16 +66,21 @@ def __init__(
                 raise ValueError("Invalid action='%s'" % self.action)
         else:
             self.limit_price = limit_price
+        # Use current time of execution if timestamp is not specified.
+        if timestamp:
```
Comment (PomazkinG) [inline defi/dao_cross/order.py:70]: nit: maybe explicitly say `if timestamp is not None`?

**Review by gpsaggese — COMMENTED** [review]
LGTM

File: defi/dao_cross/order.py [hunk 3]
```diff
@@ -27,29 +27,32 @@ def __init__(
         quote_token: str,
         action: str,
         quantity: float,
-        limit_price: float,
+        limit_price: Optional[float],
+        timestamp: Optional[pd.Timestamp],
```
[the remark this answers is not in the document]
Comment (PomazkinG) [reply defi/dao_cross/order.py:31]: thus the type hint becomes `Union[float, pd.Timestamp]`

I see that we use `timestamp = np.nan` below

**Review by PomazkinG — COMMENTED** [review]
0.05 ₽ from me

**Comment by gpsaggese** [comment]
Also some related follow up are from PR https://github.com/sorrentum/sorrentum/pull/107

@DanilYachmenev / @PomazkinG can you discuss and take care of those changes together with these?

**Pull request merged**